content/index.md

```diff
@@ -3,6 +3,135 @@ date: "2017-10-10T11:27:27-04:00"
 type: index
 ---
 
-![opencensus](/images/opencensus.png)
+<table>
+<tr>
+<td><img src="/images/opencensus.svg" width="400"></td>
+<td>
+<h1> What is OpenCensus?</h1>
 
-OpenCensus is coming soon...
+<p>A single distribution of libraries that automatically collects traces and
+metrics from your app, displays them locally, and sends them to any analysis
+tool.</p>
+</td>
+</tr>
+</table>
+
+The key features of OpenCensus include:
+
++   Standard wire protocols
+    ([TraceContext](https://github.com/TraceContext/tracecontext-spec)) and
```

Comment:
We shouldn't give a link to the spec right now as it is still an early draft.

We can instead say for now:

```
Standard wire protocols and consistent APIs for handling trace and metric data.